.github/workflows/auto-close-github-discussions.yml

```diff
@@ -0,0 +1,39 @@
+name: Auto Close Stale Discussions
+
+on:
+  schedule:
+    - cron: '0 0 * * *' # Runs daily at midnight UTC
+  workflow_dispatch: # Enables manual run from the Actions tab
+
+permissions:
+  contents: read
+  discussions: write
+
+jobs:
+  close-discussions:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '20'
+
+      - name: Install dependencies
+        run: npm install @octokit/rest @octokit/graphql
```

Comment:
Add these to dev dependencies in `package.json` and use `pnpm install` here

Reply:
Done!